sormas-backend/src/test/java/de/symeda/sormas/backend/sormastosormas/data/received/Sormas2SormasDataValidatorTest.java

```diff
@@ -22,9 +22,13 @@
 
```

Comment:
Please rename this class to `ReceivedDataProcessorTest` because it isn't testing `Sormas2SormasDataValidator` at all. If there are some validator specific tests than the processor tests should be moved to another test class.

Reply:
renamed